frontend/src/main.ts

```diff
@@ -39,6 +38,5 @@ Object.keys(Icons).forEach((key) => {
 app.use(router);
 app.use(i18n);
 app.use(pinia);
-app.use(directives);
 app.use(Components);
 app.mount('#app');
```

Comment:
The provided code looks mostly correct, but here are some slight adjustments and optimization suggestions:

### Adjustments:
1. **Import Directives**:
   - Ensure that `directives` is correctly imported. It seems like there might be a typo in the comment line.

```javascript
import directives from '@/directives';
```

2. **Remove Unnecessary Blank Line**:
   - The first blank line before `Object.keys(Icons).forEach` can be removed to improve readability.

### Optimization Suggestions:
- **Use `defineComponent()` Instead of Function Assignment**:
  - You could wrap each component import inside `defineComponent()`, which helps with TypeScript definitions more clearly.

```javascript
app.component(ComponentName, defineComponent(() => import('@/components/ComponentName.vue')));
```

- **Minimize Imports**:
  - If you're using modules for components, consider importing all at once rather than individual ones if they share a common parent directory structure.

```javascript
import * as Components from '@/components/';
```

- **Avoid Repeated Use of `app.use(`)**:
  - While `use` methods on Vue instance can be called multiple times, it's generally best practice not to redefine them unnecessarily. However, this might depend on how your application architecture is designed.

Overall, keeping these minor tweaks will make the code cleaner and potentially enhance performance slightly.